platform/state/platform_state.proto

```diff
@@ -0,0 +1,391 @@
+/**
+ * # PlatformState
+ * Messages that hold platform state in the network state.
+ *
+ * The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT",
+ * "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this
+ * document are to be interpreted as described in [RFC2119](https://www.ietf.org/rfc/rfc2119)
+ * and clarified in [RFC8174](https://www.ietf.org/rfc/rfc8174).
+ */
+syntax = "proto3";
+
+package com.hedera.hapi.platform.state;
+
+/*
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+option java_package = "com.hedera.hapi.platform.state.legacy";
+// <<<pbj.java_package = "com.hedera.hapi.platform.state">>> This comment is special code for setting PBJ Compiler java package
+option java_multiple_files = true;
+
+import "basic_types.proto";
+import "timestamp.proto";
+
+/**
+ * The current state of platform consensus.<br/>
+ * This message stores the current consensus data for the platform
+ * in network state.
+ *
+ * The platform state SHALL represent the latest round's consensus.<br/>
+ * This data SHALL be used to ensure consistency and provide critical data for
+ * restart and reconnect.
+ */
+message PlatformState {
+    /**
+     * An address book for this round.
+     * <p>
+     * This SHALL be the latest network-consensus view of consensus nodes.
+     */
+    AddressBook address_book = 1;
+
+    /**
+     * A previous address book.
+     * <p>
+     * If present, the previous address book SHALL be the address book from the
+     * most recent preceding round.
+     * <p>
+     * This is a temporary workaround until dynamic address books are supported.
+     */
+    AddressBook previous_address_book = 2;
```

Comment:
These should be removed (assuming we are going with the plan to use a temporary address book leaf prior to TSS).

Reply:
I removed both `AddressBook` fields